tolopay/src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+// import { FaUserAlt } from "react-icons/fa"
+import { BsBellFill } from "react-icons/bs"
+import { tologoSVG } from "../../Assets";
+import Profile from "../ProfileComponent/Profile"
+// import Profile from "../ProfileComponent/Profile"
+// <BsBellFill />
+// <FaUserAlt />
+
+import './Navbar.css';
+function Navbar() {
+   return (
+<>
+         <div class="navbar">
+            
+         <a href="#logo" class="logos">
+              <img src={tologoSVG} alt="logo" />
```

Comment:
```suggestion
             // <img src={tologoSVG} alt="logo" />
```

logo repeated, it would be nice if this logo is removed